src/frontend/js/widgets/Dashboard/components/DashboardBreadcrumbs/_styles.scss

```diff
@@ -19,6 +19,7 @@
   li {
     margin-right: 1rem;
     padding-right: 0.5rem;
+    height: 1.5rem;
```

Comment:
Sure? Why not use padding instead ?

Reply:
Yes the icon break the height and resulting in a broken alignement :/